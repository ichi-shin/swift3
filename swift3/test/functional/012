#!/bin/bash

# PUT Bucket logging

. ./common

echo '<?xml version="1.0" encoding="UTF-8"?>
<BucketLoggingStatus xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
  <LoggingEnabled>
    <TargetBucket>bucket2</TargetBucket>
    <TargetPrefix>access_log-</TargetPrefix>
  </LoggingEnabled>
</BucketLoggingStatus>' > $tmp.logging.xml

S3USER=tester

_s3_put /bucket
_s3_put /bucket2 -H 'x-amz-acl: log-delivery-write'

_s3_put /bucket?logging -T $tmp.logging.xml -D - | _filter_curl
