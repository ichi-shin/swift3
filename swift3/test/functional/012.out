QA output created by 012
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X PUT -H x-amz-acl: log-delivery-write http://SWIFT_HOST/bucket2... 200
> s3curl --id tester -- -X PUT -T /TMP.logging.xml -D - http://SWIFT_HOST/bucket?logging... 200
HTTP/1.1 100 Continue

HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: text/html; charset=UTF-8
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

